api/cont.js

```diff
@@ -114,7 +114,17 @@
                 continuation
                     .extend({source: sourceFragment}, idx, sourceLength, value)
                     .evalReference();
-            });
+            };
+
+            if (typeof source.length === 'number') {
+                $.each(source, eachCallback);
+            }
+            else {
```

Comment:
It's not just on some device. It's across all devices and browser. We can see this in Google Analytics.